smoked sheep/Enemy.cpp

```diff
@@ -0,0 +1,268 @@
+#include"Enemy.h"
+#include"map.h"
+
+//�G�l�~�[�摜�̔䗦��200�~287
+
+CUSTOMVERTEX enemy[100][4] =
+{
+	{ 0.0f,  0.0f,  0.5f, 1.0f, 0xFFFFFFFF, 0.0f, 0.0f },
+	{ 64.0f, 0.0f,  0.5f, 1.0f, 0xFFFFFFFF, 1.0f, 0.0f },
+	{ 64.0f, 64.0f, 0.5f, 1.0f, 0xFFFFFFFF, 1.0f, 1.0f },
+	{ 0.0f,  64.0f, 0.5f, 1.0f, 0xFFFFFFFF, 0.0f, 1.0f },
+};
+enemy_wolf wolf[100];
+
+float Enemy_tu(enemytype a) {
+	switch (a)
+	{
+	case nomal:
+		return 0.f / 2048.f;
+	case speed:
+		return 800.f / 2048.f;
+	case strong:
+		return 0.f / 2048.f;
+	case hide:
+		return 0.f / 2048.f;
+	}
+}
+
+float Enemy_tv(enemytype a) {
+	switch (a)
+	{
+	case nomal:
+		return 0.f / 2048.f;
+	case speed:
+		return 0.f / 2048.f;
+	case strong:
+		return 287.f / 2048.f;
+	case hide:
+		return 574.f / 2048.f;
+	}
+}
+
+float EnemyWidth_tu() {
+	return 200.f / 2048.f;
+}
+float EnemyHeight_tv() {
+
+	return 287.f / 2048.f;
+}
+void FormatEnemy_tu()
+{
+	enemy[enemy_number][0].tu = 0.f / 2048.f;
+	enemy[enemy_number][1].tu = EnemyWidth_tu();
+	enemy[enemy_number][2].tu = EnemyWidth_tu();
+	enemy[enemy_number][3].tu = 0.f / 2048.f;
+}
+void FormatEnemy_tv() {
+	enemy[enemy_number][0].tv = 0.f / 2048.f;
+	enemy[enemy_number][1].tv = 0.f / 2048.f;
+	enemy[enemy_number][2].tv = EnemyHeight_tv();
+	enemy[enemy_number][3].tv =  EnemyHeight_tv();
+}
+
+void SettingEnemy_tv(enemytype a)
```

Comment:
関数名が
```suggestion
void SetEnemy_tv(enemytype a)
```
のほうが文法的に正しいと思います

Reply:
訂正しました。